packages/terra-toggle-section-header/docs/README.md

```diff
@@ -0,0 +1,36 @@
+# Terra Toggle Section Header
+
+Toggle section header component that transitions content in and out with a click.
+
+## Getting Started
+
+- Install with [npmjs](https://www.npmjs.com):
+  - `npm install terra-toggle-section-header`
+
+## Usage
+
+```jsx
+import React from 'react';
+import ToggleSectionHeader from 'terra-toggle-section-header';
+
+<ToggleSectionHeader
+  title="Section Header Title"
+  level={3}
+  isInitiallyOpen
+>
+  <div>Hello, World!</div>
+</ToggleSectionHeader>
+```
+
+## Component Features
+
+ * [Cross-Browser Support](https://github.com/cerner/terra-core/wiki/Component-Features#cross-browser-support)
+ * [LTR/RTL Support](https://github.com/cerner/terra-core/wiki/Component-Features#ltr--rtl-support)
+
+ 
+<!-- Uncomment supported features.
+ * [Responsive Support](https://github.com/cerner/terra-core/wiki/Component-Features#responsive-support)
+ * [Mobile Support](https://github.com/cerner/terra-core/wiki/Component-Features#mobile-support)
+ * [Internationalization Support](https://github.com/cerner/terra-core/wiki/Component-Features#internationalization-i18n-support)
+ * [Localization Support](https://github.com/cerner/terra-core/wiki/Component-Features#localization-support)
```

Comment:
we should uncomment localization support as well as responsive/mobile support

Reply:
### localization
* What [translations](https://github.com/cerner/terra-core/wiki/Component-Features#localization-support) are being performed?

Reply:
I'm fine leaving these commented out.